mirdata/utils.py

```diff
@@ -29,11 +29,45 @@
 import hashlib
 import os
 import json
+import types
 
 
 MIR_DATASETS_DIR = os.path.join(os.getenv('HOME', '/tmp'), 'mir_datasets')
 
 
+class Track(object):
+    def __repr__(self):
+        properties = [v for v in dir(self.__class__) if not v.startswith('_')]
+        attributes = [
+            v for v in dir(self) if not v.startswith('_') and v not in properties
+        ]
+
+        repr_str = "Track(\n"
+
+        for attr in attributes:
+            val = getattr(self, attr)
+            if isinstance(val, str):
```

Comment:
should this be `if not isinstance`?

Reply:
if you support py2 still (do you?) you need to check baseinstance for `basestring`

Reply:
I think it's right - I'm formatting strings with visible `"` around the values when printing